File: SDS/numberTheory/num_I_2725.cpp
#include<iostream>

using namespace std;

int gcd(int a, int b)
{
    if(a < b)
    {
        int tmp = a;
        a = b;
        b = tmp;
    }
    
    while(b != 0)
    {
        int tmp = b;
        b = a % b;
        a = tmp;
    }
    
    return a;
}

int main()
{
    int T;
    
    scanf("%d", &T);
    
    while(T--)
    {
        int N;
        int count = 0;
        scanf("%d", &N);
        
        for(int i=1; i<N+1; i++)
        {
            for(int j=i+1; j<N+1; j++)
            {
                if(gcd(j,i) == 1)
                    count += 1;
            }
        }
        
        printf("%d\n", count*2+3);
    }
}

File: SDS/numberTheory/num_I_2725.py
def gcd(a, b):
    if a < b:
        a, b = b, a
        
    while b != 0:
        a, b = b, (a % b)
    return a

T = int(input())

ans = [ 0 for _ in range(1001)]
ans[0] = 0
ans[1] = 2

for i in range(2, 1001):
    count = 0
    for j in range(1, i+1):
        if gcd(j, i) == 1:
            count += 1
    ans[i] = ans[i-1] + count

for t in range(T):
    N = int(input())
    
    print(ans[N]*2 + -1)
                
            